storybook/stories/chart/areachart/AreaChart.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { format, parseISO, subDays } from "date-fns";
+import { ComponentMeta } from '@storybook/react';
+import { Area, AreaChart, CartesianGrid, ResponsiveContainer, Tooltip, XAxis, YAxis } from '../../../../src';
+import { TimeSeriesData } from '../../data';
+
+export default {
+  title: 'Recharts/Chart/Area Chart',
+  component: AreaChart,
+} as ComponentMeta<typeof AreaChart>;
+
+const data: TimeSeriesData[] = [];
+for (let num = 30; num >= 0; num--) {
+  data.push({
+    date: subDays(new Date(), num).toISOString().substring(0, 10),
+    value: 1 + Math.random(),
+  });
+}
+
+export const Simple = {
+  render: (args: any) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <AreaChart data={data}>
+          <defs>
+            <linearGradient id="color" x1="0" y1="0" x2="0" y2="1">
+              <stop offset="0%" stopColor="#2451B7" stopOpacity={0.4} />
+              <stop offset="75%" stopColor="#2451B7" stopOpacity={0.05} />
+            </linearGradient>
+          </defs>
+
+          <Area dataKey="value" stroke="#2451B7" fill="url(#color)" />
+
+          <XAxis
+            dataKey="date"
+            axisLine={false}
+            tickLine={false}
+            tickFormatter={(str) => {
+              const date = parseISO(str);
+              if (date.getDate() % 7 === 0) {
+                return format(date, "MMM, d");
+              }
+              return "";
+            }}
+          />
+
+          <YAxis
+            dataKey="value"
+            axisLine={false}
+            tickLine={false}
+            tickCount={8}
+            tickFormatter={(number) => `$${number.toFixed(2)}`}
+          />
```

Comment:
A story about the AreaChart should not show any complicated behaviour of other components, but focus only on the AreaChart.

These are great examples, but they should be documented in the YAxis and XAxis stories.
```suggestion
```